src/courseware/course/sequence/sequence-navigation/SequenceNavigation.test.jsx

```diff
@@ -75,25 +75,25 @@ describe('Sequence Navigation', () => {
     const onNavigate = jest.fn();
     render(<SequenceNavigation {...mockData} {...{ onNavigate }} />);
 
-    const unitButtons = screen.getAllByRole('tab');
+    const unitButtons = screen.getAllByRole('tabpanel');
     expect(unitButtons).toHaveLength(unitButtons.length);
     unitButtons.forEach(button => fireEvent.click(button));
-    expect(onNavigate).toHaveBeenCalledTimes(unitButtons.length);
+    expect(onNavigate).toHaveBeenCalledTimes(unitButtons.length - 2);
```

Comment:
[nit]: Is the magic number 2 the number of Prev/Next buttons? If so, I suggest creating a constant for this that will be based on the number (length) of Prev/Next buttons and use it here.
What do you think?

Reply:
Fixed